api/handlers.go

```diff
@@ -10,9 +10,11 @@ import (
 	"time"
 
 	"github.com/NebulousLabs/skynet-accounts/database"
+	"github.com/NebulousLabs/skynet-accounts/hash"
 	"github.com/NebulousLabs/skynet-accounts/jwt"
 	"github.com/NebulousLabs/skynet-accounts/metafetcher"
 	"github.com/julienschmidt/httprouter"
+	"github.com/sirupsen/logrus"
```

Comment:
why not use our own lib?

Reply:
I like the standard severity levels. Currently `accounts` is running with `trace` level logging but I want to be able to bring that up to `info` and only switch to `debug` and `trace` when I need to.

Reply:
Switching the logging library might be a good idea, actually. I would just not go for the lib we wrote for `skyd`. Instead I would go for a structured logging library, such as [zap](https://github.com/uber-go/zap) or [zerolog](https://github.com/rs/zerolog).